composer.json

```diff
@@ -1,25 +1,23 @@
 {
-  "name": "paytm/paytm-pg",
-  "description": "Paytm PHP Library",
-  "keywords": ["Paytm","Client","SDK","PHP"],
-  "authors": [
-    {
-      "name": "Sakshi Jain",
-      "role": "Developer"
+    "name": "paytm/paytm-pg",
+    "description": "Paytm PHP Library",
+    "keywords": ["Paytm", "Client", "SDK", "PHP"],
+    "authors": [{
+        "name": "Sakshi Jain",
+        "role": "Developer"
+    }],
+    "homepage": "https://developer.paytm.com/docs",
+    "require": {
+        "php": ">=7.2.0",
+        "ext-json": "*",
+        "ext-curl": "*",
+        "ext-openssl": "*",
+        "netresearch/jsonmapper": "*",
+        "monolog/monolog": "*"
```

Comment:
@muditgulgulia Don't know whether this repo's owner will merge your code or not but can you please explain the usage of `*` instead of latest `package version number`?